channels-rke2.yaml

```
@@ -325,6 +325,33 @@ releases:
       rke2-canal:
         repo: rancher-rke2-charts
         version: v3.21.4-build2022020801
+  - version: v1.21.10-rc2+rke2r2
+    minChannelServerVersion: v2.6.4-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: &serverArgs-v1-21-10-rc2-rke2r2
+      <<: *serverArgs-v1-21-10-rke2r1
+    agentArgs: &agentArgs-v1-21-10-rc2-rke2r2
+      <<: *agentArgs-v1-21-10-rke2r1
+    charts: &charts-v1-21-10-rc2-rke2r2
+      <<: *charts-v1-21-10-rke2r1
+      rke2-canal:
+        repo: rancher-rke2-charts
+        version: v3.21.4-build2022020801
```

CI actually is helpful in this case! I would've never seen this (one of the 0's should be a 2 apparently). 
```suggestion
        version: v3.21.4-build2022022801
```